src/frontend/utils/converse.ts

```diff
@@ -29,8 +29,11 @@ export const converseMounter = () => {
         hide_muc_participants: true,
         jid: xmpp.jid,
         modtools_disable_assign: true,
+        muc_instant_rooms: false,
         root: document.querySelector(containerName),
+        show_client_info: false,
```

Comment:
it seems you are doing 2 more changes on top of theme change. Can you explain in description and commits?

Reply:
Done